projecttacoma/bulk-submit-demo
# Filters tab

## Summary
Added a component which populates the Filters tab of the Data Requirements Viewer. 

### New Behavior
Just like the JSON tab, the filters tab will now populate on Measure ID change.

### Code Changes
Created the DataRequirementsFilter Component
Broke out the DataRequirementsJSON Component from the panel
Updated file directory structure to group panel components together
added testing

## Testing Guidance
- run `npm test`
- Make sure your test server is populated with measures and start it up
- run `npm run dev` in the bulk submit demo
- navigate to http:localhost:3002
- Select the filters tab and ensure it shows "No Data"
- Select a measure and ensure the filters tab populates
- Select a different measure and ensure the tab updates
